<?xml version="1.0"?>

<launch>
    <include file="$(find rt2_first_assignment)/launch/simulation_gmapping.launch"/>
    <include file="$(find rt2_first_assignment)/launch/move_base.launch"/>    
    <node pkg="rt2_first_assignment" type="mode.py" name="mode" output="screen" required="true" launch-prefix="xterm -e"/> 
</launch>
